src/compiler/browser/browser-scope-css.ts

```diff
@@ -0,0 +1,23 @@
+import * as d from '../../declarations';
+import { catchError } from '@utils';
+import { getScopeId } from '../style/scope-css';
+import { scopeCss as shadowCss } from '../../utils/shadow-css';
+
+
+export const scopeCss = async (code: string, tagName: string, opts: d.ScopeCssOptions = {}) => {
```

Comment:
a lot of functions with the same name, can get tricky to maintain in the future

Reply:
Yeah the shadow-css.ts file and fn needs to be renamed.